webrtc.html

```diff
@@ -399,23 +399,31 @@
 
         <ol>
           <li>
-            <p>Append an object representing the current call being handled
-            (i.e. function name and corresponding arguments) to the
-            <code>operations</code> array.</p>
+            <p>Let <var>p</var> be the result of transforming
+            <code>operations</code> by a fulfillment handler that runs the
```

Comment:
What does "operations" refer to? If it is the internal variable referred to above, then that sentence needs rephrasing.